logging/import-logs/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+backoff==2.2.1
+pytest==7.0.1
```

Comment:
Consider using latest versions of all packages in initial sample release, e.g. pytest 7.4.2

Reply:
Updated